pypom/selenium_driver.py

```diff
@@ -0,0 +1,133 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from zope.interface import (
+    implementer,
+    Interface,
+)
+
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver import (
+    Remote,
+    Opera,
+)
+
+from .interfaces import IDriver
+from .driver import registerDriver
+
+
+class ISelenium(Interface):
+    """ Marker interface for Selenium"""
+
+
+@implementer(IDriver)
+class Selenium(object):
+
+    def __init__(self, driver):
+        self.driver = driver
+
+    def wait_factory(self, timeout, poll_frequency=0.5, ignored_exceptions=None):
+        """Returns a WebDriverWait like property for a given timeout.
+
+        :param timeout: Timeout used by WebDriverWait like calls
+        :param poll_frequency: Poll frequency
+        :param ignored_exceptions: Ignored exceptions by WebDriverWait
+        :type timeout: int
+        :type poll_frequency: float
+        :type ignored_exceptions: list
+        """
+        return WebDriverWait(self.driver, timeout, poll_frequency=poll_frequency, ignored_exceptions=ignored_exceptions)
```

Comment:
It looks like we're still only using the `timeout` so I don't think we need the additional keyword arguments. This is just a convenience method too, users would be encouraged to create their own wait objects if they wanted more flexibility.